core/runtime/src/main/java/io/quarkiverse/langchain4j/audit/AuditSourceInfo.java

```diff
@@ -0,0 +1,77 @@
+package io.quarkiverse.langchain4j.audit;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+
+import io.quarkiverse.langchain4j.runtime.aiservice.AiServiceMethodCreateInfo;
+
+/**
+ * Contains information about the source of an audit event
+ */
+public final class AuditSourceInfo {
```

Comment:
As this is something users are onlt going to be consuming, I'd rather it be an interface

Reply:
Good thought.

Reply:
done